app/process/roads_bikelanes/bikelanes/Bikelanes.lua

```diff
@@ -65,13 +66,10 @@ function Bikelanes(object)
       if category ~= nil then
         local results = { _infrastructureExists = true, category = category, offset = sign * RoadWidth(tags) / 2 }
 
+        results.oneway = cycleway.oneway
```

Comment:
Das müsste hier reichen, oder? Der Rest kann weg.

```suggestion
        results.oneway = cycleway.oneway or InferOneway(category)
```